src/middlewared/middlewared/client/client.py

```diff
@@ -1,24 +1,31 @@
+from collections import defaultdict, namedtuple
```

Comment:
Since you're already here, can we fix the imports in this file? We _try_ to follow the common paradigm of 

1. python stdlib imports go first
--- (import blah) goes at top
--- (from blah import blah) goes next
--- use any `try: except ImportError` last
2. 3rd party modules get imported next with 1 space between the stdlib imports

3. our modules get imported with 1 space between the 3rd party modules

For example
```
import time
import os
from collections import defaultdict
try:
    from collections.abc import Callable
except ImportError:
    from collections import Callable

import psutil
from pyroute2 import Context

from middlewared.service import CallError
from . import ejson as json

Reply:
Thanks, I wasn't aware of the convention.

FWIW, I was able to get _close_ using isort with some customization:
```
$ sections=FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
$ no_lines_before=FUTURE,STDLIB,LOCALFOLDER
$ isort client.py
```
Then removed some unwanted newlines.